packages/cli/create-strapi-app/src/utils/parse-to-chalk.ts

```diff
@@ -0,0 +1,22 @@
+import chalk from 'chalk';
+
+const supportedStyles = {
+  magentaBright: chalk.magentaBright,
+  blueBright: chalk.blueBright,
+  yellowBright: chalk.yellowBright,
+  green: chalk.green,
+  red: chalk.red,
+  bold: chalk.bold,
+  italic: chalk.italic,
+};
+
+export default function parseToChalk(template: string) {
```

Comment:
why do we need this ?

Reply:
We're returning CLI introduction message from the cli-api config endpoint, and we passed it back with some color templating that we need to parse and translate to chalk color fn()s.

![Screenshot 2024-06-03 at 09 55 44](https://github.com/strapi/strapi/assets/14006828/ae0faec6-ff55-486b-a14f-6c56351579a9)

Reply:
the encoding could be directly sent from the api so you don't have to parse it 🤔

Reply:
You mean using ANSI encoding directly? I haven't thought about it since we wanted to use chalk for text color and formatting.

Reply:
Well, that is correct. We can still do it if we have the time BEOD